doc/srv6/srv6_vpn.md

```diff
@@ -0,0 +1,284 @@
+# SRv6 VPN HLD
+
+# Overview
+
+Alibaba is deploying SRv6 VPN with SONiC based white box router at its edge of the network. The Alibaba's SRv6 VPN use case has been described in muliple conferences. The basic idea follows RFC 8986, which we use SRv6 as a network programming framework for applications.  VPN information is used as service anchor points for identifying service instances in the service plane. SRv6 policy could be used to provide paths to let applications reach service instances. The following diagram provides a summary on what Alibaba would like to achieve via SRv6 VPN.
```

Comment:
you have policy RFC mentioned below some-where but overview is talking only about VPN. Please uplift text in overview to include them

Reply:
Policy is mentioned above, right?

"SRv6 policy could be used to provide paths to let applications reach service instances. "